subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1090,7 +1158,10 @@ static int flush_storage(void)
 	return stream_flash_buffered_write(&stream, NULL, 0, true);
 }
 
-static int process_buffer(uint8_t *buf, size_t len)
+#if !defined(CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_CUSTOM)
+static
+#endif
+int nrf_cloud_pgps_process_update(uint8_t *buf, size_t len)
```

Comment:
if `(index.dl_offset != 0)` there is no check for `buf == NULL`

Reply:
there's no check for `buf == NULL` for the other case either.  I'll add a check.